toolchains/nodejs/nodejs.bzl

```diff
@@ -105,3 +119,18 @@ def nixpkgs_nodejs_configure(
     )
 
     native.register_toolchains("@{}_toolchain//:nodejs_nix".format(name))
+
+def nixpkgs_nodejs_configure_platforms(
+  name = "nixpkgs_nodejs",
+  **kwargs,
+):
+    for nix_platform, bazel_platform in _nodejs_nix_to_bazel_platforms_map.items():
+        # get the constaints defined under rules_nodejs PLATFORMS variable
+        constraints = PLATFORMS[bazel_platform].compatible_with
+        nixpkgs_nodejs_configure(
+            name = "{}_{}".format(name, bazel_platform),
+            nix_platform = nix_platform,
+            exec_constraints = constraints,
+            target_constraints = constraints,
+            **kwargs,
```

Comment:
@aherrmann 
I think I know what I did wrong. So in general this ^^ code can be the default path for a user. However, we should allow users to setup whatever `platform/nix_system` they want. This platforms intersection (`__nodejs_nix_to_bazel_platforms_map`) is also something that should land in the parameters list. Maybe it should be some kind of dictionary/struct with the following "schema":
```python
{
   "x86_64-linux": {
      "rules_nodejs_platform": "linux_amd64",
      "exec_constraints": [
         ...
      ],
      "target_constraints": [
         ...
      ]
   }
   "aarch64-linux": {
      "rules_nodejs_platform": "linux_arm64",
      "exec_constraints": [
         ...
      ],
      "target_constraints": [
         ...
      ]
   }
}
```
I mean it could be something similar to [PLATFORMS from rules_nodejs](https://github.com/bazelbuild/rules_nodejs/blob/stable/nodejs/private/toolchains_repo.bzl#L20) but it should include nixpkgs `system` value. By default, rules_nixpkgs code can generate configurations for the intersection that I've presented but it should also allow users to define their own setups (whatever it can be...) as we allow to define custom `nix_file` (which can contain anything).